code_annotations/base.py

```diff
@@ -143,8 +153,8 @@ def _configure_group(self, group_name, group):
         """
         self.groups[group_name] = []
 
-        if not group:
-            raise TypeError('Group "{}" has no annotations!.'.format(group_name))
+        if not group or len(group) == 1:
+            raise ConfigurationException('Group "{}" must have more than one annotation.'.format(group_name))
```

Comment:
Any particular reason you can't have a group with only 1 annotation?

Reply:
The only purpose of a group is to force annotations to be present together, though there may be a case to be made that the presentation layer can make use of it. So it seems like any case of a group of one would likely be a misconfiguration. I'm open to other viewpoints, though, since I'm only really familiar with the PII use case.

Reply:
I can see someone putting in a group that will later have more than one annotation in it, but wanting to test it or run it for a while with just 1.  But not a huge deal either way.